source/clear-linux/guides/maintenance/kernel-modules-dkms.rst

```diff
@@ -315,20 +315,16 @@ Examples
 Related topics
 **************
 
-* `Dynamic Kernel Module System (DKMS) project on GitHub <https://github.com/dell/dkms>`_
+* `Dynamic Kernel Module System (DKMS)`_
 
 * `Dell Linux Engineering Dynamic Kernel Module Support: From Theory to Practice <https://www.kernel.org/doc/ols/2004/ols2004v1-pages-187-202.pdf>`_
 
 * `Linux Journal: Exploring Dynamic Kernel Module Support <https://www.linuxjournal.com/article/6896>`_
 
+.. _mixer: https://clearlinux.org/features/mixer-tool
```

Comment:
Please update to :ref:`mixer` . Link used ref to deprecated feature page.

Reply:
Nice catch!